README.md

```diff
@@ -1,10 +1,44 @@
 # Project Template
```

Comment:
Should we change this, possibly to the title of our project (i.e. "Blue Guppies - Clean Calendar") or something along those lines?

Reply:
Changed it to "Clean Calendar"